tests/tests_relp.yml

```diff
@@ -47,9 +47,6 @@
             cert: "{{ __test_cert }}"
             private_key: "{{ __test_key }}"
             pki_authmode: name
-            permitted_clients:
-              - '*.client.com'
-              - '*.example.com'
```

Comment:
If these are deleted from here, where are they set?

Reply:
It is set here:  https://github.com/linux-system-roles/logging/pull/210/files#diff-1a6cdc6949743db21fb21f528511be7e51a1ebc1117b0b90568de7bbdde43552R35

And the value would be `[ "*.$( hostname -d )" ]`.

Reply:
hmm - ok - then not sure why the relp tests are failing in CI

Reply:
> hmm - ok - then not sure why the relp tests are failing in CI

Me, too... Temporarily pushed a debugging commit...

Reply:
@richm, could you please review one more time? The failure on fedora 33 host was there was no domain configured.
```
Note: if the target node has no domain configured [0], ansible_domain is set
to localdomain, although `hostname -d` returns none. It must be considered
in the test cases.
[0]
  # grep domain /etc/resolv.conf
  #
```